pool/postgres.go

```diff
@@ -234,28 +270,51 @@ func (db *PostgresDB) fetchCSRFSecret() ([]byte, error) {
 
 // persistCSRFSecret stores the bytes used for the CSRF secret in the database.
 func (db *PostgresDB) persistCSRFSecret(secret []byte) error {
+	const funcName = "persistCSRFSecret"
 	_, err := db.DB.Exec(insertCSRFSecret, hex.EncodeToString(secret))
+	if err != nil {
+		desc := fmt.Sprintf("%s: unable to persist CSRF secrets: %v",
+			funcName, err)
+		return errs.DBError(errs.PersistEntry, desc)
+	}
 	return err
 }
 
 // persistLastPaymentInfo stores the last payment height and paidOn timestamp
 // in the database.
 func (db *PostgresDB) persistLastPaymentInfo(height uint32, paidOn int64) error {
+	const funcName = "persistLastPaymentInfo"
 	tx, err := db.DB.Begin()
 	if err != nil {
 		return err
 	}
 
 	_, err = tx.Exec(insertLastPaymentHeight, height)
 	if err != nil {
-		tx.Rollback()
-		return err
+		rErr := tx.Rollback()
+		if rErr != nil {
+			desc := fmt.Sprintf("%s: unable to rollback persisting last "+
+				"payment height tx: %v", funcName, rErr)
+			return errs.DBError(errs.PersistEntry, desc)
+		}
+
+		desc := fmt.Sprintf("%s: unable to persist last payment height: %s",
+			funcName, err.Error())
```

Comment:
Why use `"%s", err.Error()` here and in the following instance, when `"%v", err` is used everywhere else?